main/src/main/java/org/mobilitydata/gtfsvalidator/report/model/FeedMetadata.java

```diff
@@ -206,7 +218,32 @@ private void loadPathwayDirectionsComponent(GtfsFeedContainer feedContainer) {
             GtfsPathway.FILENAME,
             List.of(
                 GtfsPathway::hasSignpostedAs,
-                (Function<GtfsPathway, Boolean>) GtfsPathway::hasReversedSignpostedAs)));
+                (Function<GtfsPathway, Boolean>) GtfsPathway::hasReversedSignpostedAs,
+                GtfsPathway::hasMaxSlope,
```

Comment:
I don't think these 4 tests (hasMaxSlope, hasMinWidth, hasLength and hasStairCount) go with `Pathway Directions`

Reply:
Correct - should be for 'Pathways (extra)'